index.html

```diff
@@ -16,39 +16,39 @@ <h1 class="text-center" id="workWithUs">Work with Us</h1>
       </p>
     </div>
     <div class="padded-content workwithusContent">
-      <div class="row workwithusIconsRow col-md-10">
-        <div class="col-md-3 col-xxssm-6">
+      <div class="row workwithusIconsRow iconContainer">
+        <div class="iconTile iconTile-sm">
```

Comment:
It seems like every element that uses `iconTile` uses `iconTile-sm` as well. Can we group all of the styles into one class instead?

Reply:
combined into iconTile. Removed iconTile-sm